File: app/src/main/java/at/priv/graf/zazentimer/database/SessionEntity.java
package at.priv.graf.zazentimer.database;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "sessions")
public class SessionEntity {
    @PrimaryKey(autoGenerate = true)
    public int _id;
    @NonNull
    public String name;
    @NonNull
    public String description;
}

File: app/src/main/kotlin/at/priv/graf/zazentimer/database/SessionEntity.kt
package at.priv.graf.zazentimer.database

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "sessions")
@Suppress("ConstructorParameterNaming")
data class SessionEntity(
    @PrimaryKey(autoGenerate = true)
    var _id: Int = 0,
    var name: String = "",
    var description: String = "",
)
